phrosty/pipeline.py

```diff
@@ -56,6 +53,11 @@ def __init__( self, imagepath, pointing, sca ):
         # self.psf is a object of a subclass of snappl.psf.PSF
         self.config = Config.get()
         self.temp_dir = pathlib.Path( self.config.value( 'photometry.phrosty.paths.temp_dir' ) )
+        self.keep_intermediate = self.config.value( 'photometry.phrosty.keep_intermediate' )
+        if not self.keep_intermediate:
+            self.save_dir = pathlib.Path( self.config.value( 'photometry.phrosty.paths.temp_dir' ) )
+        elif self.keep_intermediate:
+            self.save_dir = pathlib.Path( self.config.value( 'photometry.phrosty.paths.dia_out_dir' ) )
```

Comment:
I would reverse these.  `if not foo` requires one more step to understand than `if foo`.